pkg/registry/registry.go

```diff
@@ -201,27 +205,30 @@ func (r *Registry) registryHandler(rw mux.ResponseWriter, req *http.Request) {
 			cacheType = "miss"
 		}
 		metrics.MirrorRequestsTotal.WithLabelValues(registryName, cacheType, sourceType).Inc()
-		return
+		return "mirror"
 	}
 
 	// Serve registry endpoints.
 	if dgst == "" {
 		dgst, err = r.ociClient.Resolve(req.Context(), ref)
 		if err != nil {
 			rw.WriteError(http.StatusNotFound, err)
-			return
+			return ""
 		}
 	}
 
 	switch refType {
 	case referenceTypeManifest:
 		r.handleManifest(rw, req, dgst)
+		return "manifest"
 	case referenceTypeBlob:
 		r.handleBlob(rw, req, dgst)
-	default:
-		// If nothing matches return 404.
-		rw.WriteHeader(http.StatusNotFound)
+		return "blob"
 	}
+
+	// If nothing matches return 404.
+	rw.WriteHeader(http.StatusNotFound)
+	return ""
```

Comment:
Would `none`, `unknown` or similar provide better usability (read less support questions)?

Reply:
I think we should keep it as empty for now to keep things simple. If it becomes a problem we can change it in the future.